ecs_helper/cmd/pact_tags/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"os"
+	"fmt"
+	"net/http"
+	"encoding/base64"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+)
+
+func basicAuth(username, password string) string {
+	auth := username + ":" + password
+	return base64.StdEncoding.EncodeToString([]byte(auth))
+}
+
+func getPactPassword(roletoassume string, pactsecretkey string) string {
+	mysession := session.Must(session.NewSession())
+	creds := stscreds.NewCredentials(mysession, roletoassume)
+	cfg := aws.Config{Credentials: creds,Region: aws.String("eu-west-1")}
+	sess := session.Must(session.NewSession(&cfg))
+	sm := secretsmanager.New(sess)
+	secretkey := pactsecretkey
```

Comment:
Looks like this is just referencing `pactsecretkey` - could we pass it straight into `GetSecretValue()` instead rather than assigning?

Reply:
Yeah sorry, moved it out to a function and obviously missed this.